Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitClient.swift

```diff
@@ -25,6 +32,13 @@ import BraintreeCore
         self.apiClient = apiClient
         self.sepaDirectDebitAPI = SEPADirectDebitAPI(apiClient: apiClient)
         self.webAuthenticationSession =  WebAuthenticationSession()
+        super.init()
+        NotificationCenter.default.addObserver(
```

Comment:
This notification is not getting triggered on launch of the ASWebSession browser as it is in the PayPal client for some reason.

Reply:
I put debugger on casting the webAuthenticationSession.start() method to cast `presentationContextProvider` as `BTSEPADirectDebitClient` and it seems to be set properly.

Reply:
I am wondering if it is related to the fact that alert view doesn't show on launching webView as it is for PayPal module.

Reply:
I tried saving reference to `ASWebAuthenticationSession` in the `WebAuthenticationSession` class to see if it was problem of scope outside of the start function, but that didn't fix the issue.

Reply:
I found the culprit, it was this line in WebAuthenticaionSession class: `authenticationSession.prefersEphemeralWebBrowserSession = true`
It suppresses the alert view and also trigger of system notification applicationDidBecomeActive.

Reply:
So this prefersEphemeralWebBrowserSession option requests that the browser doesn't share cookies or other browsing data between the authentication session and the user's normal browser session. Whether the request is honored depends on the user's default web browser and Safari always honors the request. 

So I guess I will leave this option as well as the notification for the browser launching. Another question I had was why this option is enabled on SEPA but not on Local Payment?

Reply:
I don't have context on why `prefersEphemeralWebBrowserSession` was set to true, as I wasn't here when SEPA was added. Either it was a requirement from the SEPA team, or we just added it because we thought it was more secure.

My guess is the later.

Reply:
I vote to remove it, and follow the same pattern as PayPal and Local Payments. Great find!

Reply:
I tested it with Chrome on my device and it also behaves the same as what I observe on the simulator but I guess the behavior might be different for different browser. I want to make sure that I am clear on any business decision for both SEPA or Local Payments.